vector/src/test/java/im/vector/app/features/home/room/detail/timeline/render/ProcessBodyOfReplyToEventUseCaseTest.kt

```diff
@@ -158,6 +159,7 @@ class ProcessBodyOfReplyToEventUseCaseTest {
         // Given
         givenTypeOfRepliedEvent(isPollMessage = true)
         givenNewContentForId(R.string.message_reply_to_sender_created_poll)
+        every { fakeRepliedEvent.getClearType() } returns EventType.POLL_START.unstable
```

Comment:
Maybe we should also add a test for `EventType.POLL_END`?

Reply:
Done.